index.js

```diff
@@ -108,30 +86,38 @@ function getRoomFromSocket(socket_obj){
 // validates socket requests
 function validateSocket(socket_obj){
     const room_id = getRoomFromSocket(socket_obj)
-
+    const room = getGameFromSet(room_id)
```

Comment:
technically dont want const here
you alter call room.addPlayer
which manipulates the room's player field

Reply:
`getGameFromSet` is unclear
Set isnt clear enough
I would do:
`getGameObjFromRoomId`
or `getGameObj()`